### ISSUE TITLE
Add detection for Server 2019

### ISSUE THREAD

**Issue by glennsarti** [puppetlabs/puppetlabs-facts#12]
Now that Windows Server 2019 is released, the detection in https://github.com/puppetlabs/puppetlabs-facts/blob/master/tasks/powershell.ps1#L30-L40 should include this.

**Comment by donoghuc** [comment]
Is using something like `(gwmi win32_operatingsystem).caption` unreliable for generating these facts in the "real world". It seems to work fairly well for our vmpooler instances. 
```
PS C:\cygwin64\home\Administrator> (gwmi win32_operatingsystem).caption
Microsoft Windows Server 2016 Standard
```
```
PS C:\cygwin64\home\Administrator> (gwmi win32_operatingsystem).caption
(gwmi win32_operatingsystem).caption
Microsoft Windows Server 2019 Standard
```

**Comment by donoghuc** [comment]
Resolved in `0.5.1` release. Thanks!
